### ISSUE TITLE
The "how to use" is way too confusing 

### ISSUE THREAD

**Issue by bascaraba** [twisted89/Manor-Lords-Save-Load-Mod#10]
Please dumb it down some more

**Comment by GrimHansl** [comment]
Hi @bascaraba, try to follow [my video instruction](https://youtu.be/kbUpHGNKVN4)

**Comment by bascaraba** [comment]
Hi , i did manage to get it working yesterday , like in your video the coat of arms and lord's face gets changed when you load  , that wouldn't be a problem if it was all but in my experience after i load the game like 2-3 times 2 trees become 100 times the size which is immersive breaking to the fullest , do you want me to give you a video of it?

**Comment by GrimHansl** [comment]
Well I played for a few hours and after save-load I had problems with textures and the game was crashing. I don't think there's anything can do about it except wait for the release (╯︵╰,)

**Comment by bascaraba** [comment]
sad man i really wanted to build a nice village
